build/AzurePipelinesTemplates/WindowsAppSDK-SetupBuildEnvironment-Steps.yml

```diff
@@ -147,6 +147,14 @@ steps:
     arguments: -Path $(Build.SourcesDirectory)\dev\common\TerminalVelocityFeatures-AppNotifications.xml -Channel $(channel) -Language C++ -Namespace Microsoft.Windows.AppNotifications -Output $(Build.SourcesDirectory)\dev\common\TerminalVelocityFeatures-AppNotifications.h
     workingDirectory: '$(Build.SourcesDirectory)'
 
+- task: powershell@2
+  displayName: 'Create CallingPreviewSupport TerminalVelocity features'
+  inputs:
+    targetType: filePath
+    filePath: tools\TerminalVelocity\Generate-TerminalVelocityFeatures.ps1
+    arguments: -Path $(Build.SourcesDirectory)\dev\common\TerminalVelocityFeatures-AppNotifications.xml -Channel $(channel) -Language C++ -Namespace Microsoft.Windows.CallingPreviewSupport -Output $(Build.SourcesDirectory)\dev\common\TerminalVelocityFeatures-CallingPreviewSupport.h
```

Comment:
If -Path=...AppNotifications.xml why -Namespace=CallingPreviewSupport?
-Namespace Microsoft.Windows.CallingPreviewSupport
Shouldn't that be Microsoft.Windows.AppNotifications?

ThereTerminalVelocityFeatures-AppNotifications.xml and .h already exist. Why a new .h given the existing .xml?

Reply:
Yup, a little weird that this was originally created as two different features from the same .xml but into separate .h files. We need to clean up our guidance and documentation for the future.